.github/workflows/release.yml

```diff
@@ -0,0 +1,185 @@
+name: Release CLI Plugins
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '16.x'
```

Comment:
@shafeeqd959 Just wanted to confirm that are we going with node V16.x.? Because I remember we decided to support V14.x and above.

CC @netrajpatel

Reply:
Yes, we should support from 14.x onwards and that should be tested from the test suite itself. This is an environment that we setup for the release.

Reply:
Why not V14.x here as well.? To make sure it works everywhere till the moment it publishes like a checkpoint.?

CC @netrajpatel

Reply:
@antonyagustine , it's a valid point, this might not be the place to test these things, rather we should have tested all the major versions which we support before the code reach here. Let's see if that is not happening, then will reduce it to 14. thanks